.gitignore

```diff
@@ -1,2 +1,5 @@
 venv
-*.pyc
\ No newline at end of file
+*.pyc
+*.csv
+test.py
```

Comment:
we should submit changes to .gitignore globally on the master branch - otherwise there will a ton of merge conflicts when we try to merge branches into master - so please reset the .gitignore to the original state and then do a commit to the offer_view branch

Reply:
Done.